packages/core/helper-plugin/package.json

```diff
@@ -39,13 +46,13 @@
     "watch": "yarn create:index && cross-env NODE_ENV=development webpack-cli -w"
   },
   "dependencies": {
-    "axios": "1.2.2",
+    "@strapi/design-system": "1.4.1",
+    "@strapi/icons": "1.4.1",
```

Comment:
@joshuaellis This is what I meant :) Most dependencies show up as both: dependencies and peerDependencies :)

Reply:
Sorry I merged the wrong way round.